src/internal/observable/forkJoin.ts

```diff
@@ -21,7 +20,7 @@ export function forkJoin<A extends readonly unknown[], R>(
 // forkJoin(a, b, c)
 /** @deprecated Use the version that takes an array of Observables instead, Details https://rxjs.dev/deprecations/array-argument */
 export function forkJoin<A extends readonly unknown[]>(...sources: [...ObservableInputTuple<A>]): Observable<A>;
-/** @deprecated resultSelector is deprecated, pipe to map instead */
+/** @deprecated Use the version that takes an array of Observables instead, Details https://rxjs.dev/deprecations/array-argument */
```

Comment:
This link appears to be 404. @niklas-wortmann ... I assume this is just something that needs to be deployed still?

Either way, I don't think that's a blocker at this point.